src/dyz.c

```diff
@@ -10,6 +12,11 @@ int main(int _argc, char** _argv)
     argc = _argc;
     argv = _argv;
 
+#ifdef LUASEARCHPATH
```

Comment:
To me, the preferred way would be to compile the bytecode and include that in a binary. OTOH I see the value of wanting to run the Lua code through an interpreter as a quicker way of testing modifications.

We should probably support both of these modes. I can see myself doing quick test iterations by running the code through the interpreter. But for the target hardware it's, at least for me, far simpler to generate the bytecode and cross-compile for the target arch.